src/service/action-impl.js

```diff
@@ -628,7 +690,7 @@ class ParserTokenizer {
     const value = convertValues && (s == 'true' || s == 'false') ?
         s == 'true' : s;
     newIndex = end - 1;
-    return {type: TokenType.LITERAL, value, index: newIndex};
+    return {type: TokenType.ID, value, index: newIndex};
```

Comment:
It sounds like if it's `true` or `false` - this should still be `LITERAL`, right? A literal boolean value? Since we likewise have literal numeric values....